### PR TITLE
[RFR] Update Tutorial for better first contact

### PR DESCRIPTION
**Pull request by fzaninotto**
The tutorial was a bit hard to follow. I made shorter sections, added a bit more explanations, and removed non-related content.

Also, I added the ability to run an `<Admin>` with no child (just displays a confirmation screen). IMO, it provides a better developer experience for first time users, by allowing baby steps.

### CODE REVIEW

File: packages/ra-core/src/CoreAdminRouter.js [hunk 1]
```diff
@@ -93,15 +99,27 @@ export class CoreAdminRouter extends Component {
...
 
-        let childrenToRender =
-            typeof children === 'function' ? childrenFromState : children;
+        if (typeof children !== 'function' && !children) {
+            return (
+                <div style={welcomeStyles}>
+                    React-admin is properly configured.<br />
+                    Now you can add a first &lt;Resource&gt; as child of
+                    &lt;Admin&gt;.
+                </div>
```
Comment (djhi) [inline packages/ra-core/src/CoreAdminRouter.js]: Why not also add a code snipper ? Great welcome message though :+1:

Comment (fzaninotto) [reply packages/ra-core/src/CoreAdminRouter.js]: Because the code for that message is bundled to production, even though it's only useful in development. I don't want to add too much weight to the final production bundle.
